search-sentiments-map/routes/search.js

```diff
@@ -71,20 +71,20 @@ async function retrieveSearchResultFromDatastore(topic) {
  * Deletes stale data from Datastore.
  */
 async function updateSearchResults() {
-  await deleteAncientResults();
-  retrieveGlobalTrends().then(async trends => {
+  await search.deleteAncientResults();
+  const trends = await search.retrieveGlobalTrends();
+  for (let i = 0; i < trends.length; i++) {
+    await search.updateSearchResultsForTopic(trends[i].trendTopic);
     // Note: when testing ,use i < 1 to test for only one trend, and comment 
     // out `await new Promise` line to avoid 1 minute pauses.
-    for (let i = 0; i < trends.length; i++) {
-      await updateSearchResultsForTopic(trends[i].trendTopic);
-      await new Promise(resolve => setTimeout(resolve, PAUSE_ONE_MIN_MS));
-    }
-  });
+    await search.sleepForOneMinute();
+  }
 }
 
 /** 
  * Obtains the most recent global trends by querying the Datastore.
- * @return {!Array<JSON>} A JSON array of global trends and their originating countries.
+ * @return {!Array<JSON>} A JSON array of global trends and their originating
+ *     countries.
```

Comment:
```suggestion
 * countries.
```
same styling issue as in the previous PR for block comments, JSDOc comments

Reply:
See comment from Chenyu on the other comment. Section 7.4 of the Javascript style guide shows this is correct.